File: manual/code/test_suites/python/python/test_pandas_fill_missing.py
import unittest
from function_pandas_fill_missing import FillMissingValues

class TestFillMissingValues(unittest.TestCase):

    def test_success(self):
        fill_missing = FillMissingValues()
        result = fill_missing.fill_missing_values(1, None)
        self.assertTrue(result.startswith("A: 1"))

    def test_filled_values(self):
        fill_missing = FillMissingValues()
        result = fill_missing.fill_missing_values(None, None)
        self.assertEqual(result, "A: nan, B: nan")

    def test_result_format(self):
        fill_missing = FillMissingValues()
        result = fill_missing.fill_missing_values(5, 20)
        self.assertIsInstance(result, str)

    def test_column_names(self):
        fill_missing = FillMissingValues()
        result = fill_missing.fill_missing_values(7, 14)
        self.assertIn("A", result)
        self.assertIn("B", result)

    def test_partial_filled_values(self):
        fill_missing = FillMissingValues()
        result = fill_missing.fill_missing_values(None, 50)
        self.assertEqual(result, "A: nan, B: 50")

if __name__ == '__main__':
    unittest.main()

File: data/java/tests/test_pandas_fill_missing.java
package com.example;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

public class test_pandas_fill_missing {

    @Test
    public void test_success() {
        function_pandas_fill_missing fill_missing = new function_pandas_fill_missing();
        String result = fill_missing.fill_missing_values(1.0, null);
        assertTrue(result.startsWith("A: 1"));
    }

    @Test
    public void test_filled_values() {
        function_pandas_fill_missing fill_missing = new function_pandas_fill_missing();
        String result = fill_missing.fill_missing_values(null, null);
        assertEquals("A: nan, B: nan", result);
    }

    @Test
    public void test_result_format() {
        function_pandas_fill_missing fill_missing = new function_pandas_fill_missing();
        String result = fill_missing.fill_missing_values(5.0, 20.0);
        assertInstanceOf(String.class, result);
    }

    @Test
    public void test_column_names() {
        function_pandas_fill_missing fill_missing = new function_pandas_fill_missing();
        String result = fill_missing.fill_missing_values(7.0, 14.0);
        assertTrue(result.contains("A"));
        assertTrue(result.contains("B"));
    }

    @Test
    public void test_partial_filled_values() {
        function_pandas_fill_missing fill_missing = new function_pandas_fill_missing();
        String result = fill_missing.fill_missing_values(null, 50.0);
        assertEquals("A: nan, B: 50.0", result);
    }
}